tests/system_tests_edge_router.py

```diff
@@ -2521,14 +2568,18 @@ def check_address(self):
         self.num_attempts += 1
         for result in outs.results:
             if self.address in result[0]:
-                found = True
-                self.create_sndr()
-                local_node.close()
-                self.addr_timer.cancel()
-                break
+                if self.subscriber_count == 0 or (result[6] + result[7] == self.subscriber_count):
```

Comment:
Might it be safer to use the actual attribute names rather than hardcoded offsets to access the results?
Example: https://github.com/apache/qpid-dispatch/blob/master/tests/system_tests_one_router.py#L616

Reply:
What is going on in https://github.com/apache/qpid-dispatch/blob/master/tests/system_tests_one_router.py#L616 is not quite what is going on here but I added a commit which will make you happy. Please take a look.